dts/vendor/nordic/nrf54lm20a_ns_partition.dtsi

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&cpuapp_rram {
+	/*
+	 * Default NVM layout on NRF54LM20A Application MCU without BL2:
+	 * This layout matches (by necessity) that in the TF-M repository:
+	 *
+	 * 0x0000_0000 Secure image primary (512 KB)
+	 * 0x0008_0000 Protected Storage Area (16 KB)
+	 * 0x0008_4000 Internal Trusted Storage Area (16 KB)
+	 * 0x0008_8000 OTP / NV counters area (8 KB)
+	 * 0x0008_A000 Non-secure image primary (1356 KB)
+	 * 0x001D_DD00 Non-secure storage, used when built with NRF_NS_STORAGE=ON,
+	 *             otherwise unused (32 KB)
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* nRF54LM20A has 2036 kB of non-volatile memory (RRAM) but the last
+		 * 96 kB are reserved for the FLPR MCU.
```

Comment:
no, no no no, we agreed not to do this again @carlescufi

Reply:
no flpr reservation in normal builds without flpr, use full RRAM for application core